src/reducers/match.js

```diff
@@ -51,7 +51,12 @@ export default (state = initialState, action) => {
 export const getMatch = state => state.app.match;
 export const getMatchData = state => state.app.match.match;
 export const getMatchLoading = state => state.app.match.loading;
-export const getMatchPlayers = state => state.app.match.match.players;
+
+export const getMatchPlayers = (state, team = '') =>
```

Comment:
I would move this filtering to the Match component.  Also instead of using string comparison I'd probably just take an isRadiant boolean parameter.